src/native/s3select/s3select.gyp

```diff
@@ -13,7 +13,7 @@
             's3select_napi.cpp'
         ],
 	'link_settings': {
-		'libraries': ['/lib64/libboost_thread.so.1.75.0']
+		'libraries': ['/lib64/libboost_thread.so.link']
```

Comment:
This change will break the downstream build
We should choose:
1. Ask Boris to change accordingly
OR
2. Not do `RUN ln -s /lib64/libboost_thread.so.1.* /lib64/libboost_thread.so.link` and explicitly do:
```
RUN sed -i -e 's|1.75.0|1.66.0|g' $REMOTE_SOURCE_DIR/noobaa-core/app/src/native/s3select/s3select.gyp
```
When the RHEL version is 8.

Reply:
Not sure it would break ds, I think Boris would get the same symlink.
At any rate current version has a softlink from 75 to 66, which I think is the simplest solution without sed.
Hopefully next time we will switch to symlink and then we won't need to worry about version.